pkg/applicationmanager/controllers/applicationrestore.go

```diff
@@ -286,6 +302,49 @@ func (a *ApplicationRestoreController) handle(ctx context.Context, restore *stor
 		return nil
 	}
 
+	if len(restore.Spec.StorageClassMapping) >= 1 && isStorageClassMappingContainsDefault(restore) {
+		// Update the default storageclass name in storageclassmapping.
+		// storageclassMapping will have "use-default-storage-class" as destination storage class,
+		// If default storageclass need to be selected.
+		scList, err := storage.Instance().GetDefaultStorageClasses()
+		if err != nil {
+			log.ApplicationRestoreLog(restore).Errorf(err.Error())
+			a.recorder.Event(restore,
+				v1.EventTypeWarning,
+				string(storkapi.ApplicationRestoreStatusFailed),
+				err.Error())
+			return nil
+		}
+		// If more than one storageclass is set as default storageclass, update error event
+		if len(scList.Items) > 1 {
+			errMsg := "more than one storageclass is set as default on destination cluster"
+			log.ApplicationRestoreLog(restore).Errorf(errMsg)
+			a.recorder.Event(restore,
+				v1.EventTypeWarning,
+				string(storkapi.ApplicationRestoreStatusFailed),
+				errMsg)
+			return nil
+		}
+		// If more than one storageclass is set as default storageclass, update error event
```

Comment:
Comment needs to change as here no storage class is found.

Reply:
fixed